test/unit/nedap/utils/test/api.cljc

```diff
@@ -168,13 +176,20 @@
                                        form
                                        @test-result))
           (sut/expect 0 :to-change 0 :from 0 :to 1)
-          `{:type :fail, :expected (impl/meta= [0 1]), :actual (~'not (impl/meta= [0 1]))}
+          `{:type :fail, :expected (sut/meta= 0 1), :actual (~'not (sut/meta= 0 1))}
 
           (sut/expect 0 :to-change {} :from {} :to ^::test {})
-          `{:type :fail, :expected (impl/meta= [{} {}]), :actual (~'not (impl/meta= [{} {}]))}
+          `{:type :fail, :expected (sut/meta= {} {}), :actual (~'not (sut/meta= {} {}))}
 
           (sut/expect (swap! a inc) :to-change @a :from 0 :to 2)
-          `{:type :fail, :expected (impl/meta= [(deref ~'a) 2]), :actual (~'not (impl/meta= [1 2]))}))))
+          `{:type :fail, :expected (sut/meta= (~'clojure.core/deref ~'a) 2), :actual (~'not (sut/meta= 1 2))}
+
+           ;; change matcher to `=`
+          (sut/expect 0 :to-change 0 :from 0 :to 1 :with =)
+          `{:type :fail, :expected (~'= 0 1), :actual ~'(not (= 0 1))}
+
+          (sut/expect (swap! a inc) :to-change @a :from 1 :to 3 :with =)
+          `{:type :fail, :expected (~'= (~'clojure.core/deref ~'a) 3), :actual ~'(not (= 2 3))}))))
 
```

Comment:
From memory (a memory which might be outdated) I think there was something wrong with the test suite under cljs, does that ring a bell? Something related to with-redefs maybe?

With which I mean, it's worth trying to get the cljs tests to fail, as a sanity check. Also can be QAd

Reply:
Added a simple case as QA.

The with-redefs solution did not work with direct linking; but works across clj and cljs otherwise :). Happened to trigger a failing build quite often last night (on both clj and cljls side, in the with-redefs part of the test), so i'm confident it works.